content/reference/drops/time_drop.md

```diff
@@ -0,0 +1,14 @@
+# Time Drop
+
+Return attributes with time. You have ability to manage time format.
```

Comment:
Also the ability to alter date format does not come from the drop, but from the filters.